packages/cli/src/commands/env/index.ts

```diff
@@ -99,8 +98,6 @@ export default async function main(client: Client) {
       telemetry.trackCliSubcommandPull(subcommandOriginal);
       return pull(client, args);
     default:
-      output.error(getInvalidSubcommand(COMMAND_CONFIG));
-      output.print(help(envCommand, { columns: client.stderr.columns }));
-      return 2;
+      return pull(client, args.concat('--memory'));
```

Comment:
I think creating a new `exec` subcommand is the safer option here. Having inferred subcommands can really end up restricting changes we can make to `vercel env` in the future without being a breaking change.

Reply:
I think this is a good use case for no subcommands, but we can do this as a follow-up after further discussion, thanks!